book/P2C9_calib.qmd

```diff
@@ -6,7 +6,7 @@ abstract: TODO (150-200 WORDS)
 {{< include _macros.tex >}}
 :::
 
-# Calibration Measures {#sec-eval-distr-calib}
+# Calibration {#sec-eval-distr-calib}
```

Comment:
there is some R-Code and simulation in there? -> would make a good exercise

It's not that surprising though? Take exponential distribution. The hazard rate (rate of events) is constant h(t) = \lambda -> expected number of events = \lambda * t = \int_0^t \lambda = H(t)